### DISCUSSION TITLE
NextCloud AIO 4.2.0 Virus on File?

### DISCUSSION THREAD

**Discussion by armouredking** [nextcloud/all-in-one#1860] [Questions] [answered]
```
[files_antivirus] Error: Not Checked. No matching rule for response []. Please check antivirus rules configuration. File: 47090 Account: %username% Path: /%username%/files/Media/Games/Ragnarok Online/Private Servers/Client/ROenglishRE-master.zip

at 2023-01-29T16:35:01+00:00
```

Filtering out the names in log, but otherwise, what is causing this? It happens repeatedly, every 20 mins or thereabouts, looks like maybe every time it kicks off a scan? There are other zip files but they aren't flagging. ClamAV logs don't have anything else either.

**Comment by armouredking** [comment]
So, checking the file elsewhere, it appears to be a false positive signal. Is there some built in way I'm missing for telling ClamAV to ignore in AIO, or is the repeated log entry just a fact of life? I see in advanced there is a rule configuration section, I'm not sure I want to create a "Not Checked" rule though because that might let something through. Creating one based on file path also doesn't seem great, though I suppose that does accomplish the basic ask.

**Comment by szaimen** [comment, marked as answer]
Hi, see https://github.com/nextcloud/files_antivirus/issues/223

With the next files_antivirus update, it should allow to more easily debug things, IIRC.
